skins/oasis/js/Tracking.js

```diff
@@ -512,68 +523,10 @@ jQuery(function($){
 
 	/** top-nav **/
 
-	(function() {
+	(function () {
 		var category = 'top-nav';
 
-		$('#WikiaHeader').on('mousedown', 'a', function(e) {
```

Comment:
Removed this tracking b/c `#WikiaHeader` is removed in favor of the new Global Nav